### ISSUE TITLE
CentOS 7.1 can't resolve all deps for nginx install

### ISSUE THREAD

**Issue by arimus** [jdauphant/ansible-role-nginx#83]
os version: 7.1.1503 (OpenLogic)

You'll see that gperftools relies on libunwind and there is an error resolving the dep:

```
...
msg: Error: Package: gperftools-libs-2.4-5.el7.x86_64
Requires: libunwind.so.8
```

libunwind is currently in the CR repo. There's a thread that talks about it here: 

https://www.centos.org/forums/viewtopic.php?f=47&t=55325

Enabling the "cr" repo resolves the problem for manual installs (e.g. yum-config-manager --enable cr), but this module still needs to enable the repo in the task (tasks/installation.packages.yml, line 13) to not fail:

```
- name: Install the nginx packages
  yum: name={{ item }} state=present disablerepo='*' enablerepo={{ "nginx," if nginx_official_repo else "" }}{{ yum_epel_repo }},{{ yum_base_repo }},cr
  with_items: nginx_redhat_pkg
  when:  nginx_is_el|bool
  tags: [packages,nginx]
```

**Comment by jdauphant** [comment]
Hi @arimus,
Thanks for the bug report.
Do you know if adding the "cr" repository can break compatibility in previous version of the OS ?

**Comment by arimus** [comment]
Per the centos site, it looks like the CR repo has been around for awhile
(centos 5 at least) to make next release packages available before they are
formally issued as complete.

The issue with enabling it may be that you install newer versions of
packages than desired. It's a bad position they put us in, releasing the
one without the other. Not sure if this issue exists in more than one
centos release either.

Probably best to only do this for affected releases to minimize potential
for newer than expected versions of things making their way to prod. For
CentOS 7 at least, there are no other options it seems, save possible
locking to an old version of nginx.

Perhaps just a var to enable the CR repo, so that there is a fix for those
having the problem on certain releases, without potentially impacting
everyone else?

**Comment by jdauphant** [comment]
Ok, thanks,
Can you submit a pull request that correct that for CentOS 7 ?

**Comment by philipwigg** [comment]
This was a bug in CentOS 7.1 and not really the fault of your Ansible role. It was fixed in CentOS 7.2 anyway.

https://bugzilla.redhat.com/show_bug.cgi?id=1289073

I think you could close this issue.

**Comment by jdauphant** [comment]
Ok thanks
